webpack.config.client.js

```diff
@@ -245,7 +263,6 @@ module.exports = ({
        */
       new CompressionPlugin({
         algorithm: 'gzip',
-        filename: '[path].gz[query]',
```

Comment:
using default naming removes the error:

```
ERROR in Conflict: Multiple assets emit different content to the same filename static/js/.gz
```

Reply:
Is this what removes some the hyphens earlier in the PR?

Reply:
I believe so. The default naming pattern is `[path][base].gz`

The chunks it produces for the mundo front page look like this:
```
framework.99d8d4dd.js
main-49d0a293.eac2e5ed.js
main-f82e0cd2.918392e9.js
main-e019dbda.5e6b9307.js
main-d91a9049.08572402.js
moment-lib.1b477054.js
commons-d181e811.7bc61060.js
commons-9928543c.88ab41da.js
commons-2940a8bc.88dbef09.js
commons-5e5cb6fb.e3250571.js
commons-49d0a293.a29e3da8.js
commons-0bc0478e.c7d66984.js
commons-229eafb5.d0dbbded.js
commons-3e10bc16.426a0dd3.js
commons-e019dbda.4f58e7b0.js
commons-3a4fa680.104ded9e.js
shared-6sI%2BDPbHATTXcp9U1qboQTGopM%3D.8845723f.js
shared-IWsrdoKm4A6gjoTLJxEjFp%2B8O0Y%3D.6cd37c47.js
shared-QS97oh2%2BNXd%2BnBZOuYERkd1RFEI%3D.e58d8d8d.js
shared-WhQoVvI2P8YOXg8QEMayWTvnGk%3D.cb779d54.js
shared-HSEfoBeEeeochntvpuL5VPDmWM%3D.51fb41d6.js
shared-2JZTQ%2BSosd7jQIgcihukspH%2BrU%3D.367654d1.js
shared-uAJRYHpKS72mDv4DjjjpiXfYw10%3D.6d8209f2.js
shared-9ODfrhqMHPGD9BIVn1rDov175TU%3D.b891d8f5.js
shared-1TA1VDJWP4NmZHwhSwt54tJkwY%3D.6fcac5da.js
shared-CY2y9BfN%2BPXvezqGWGRRAGUYHEg%3D.ad912788.js
FrontPage.86cd3b34.js
mundo.fc293624.js
```

whereas previously it looked like
```
framework-f734b0c6.d6502c7d.js
main-d0ae3f07.a623de1a.js
main-7d359b94.8de2074c.js
main-f71cff67.c136dd0e.js
main-88a3c260.02eeac5f.js
moment-lib-87d47d0c.19e78cfd.js
commons-31ecd969.90563031.js
commons-253ae210.f7e7d114.js
commons-0b7c41be.9c418545.js
commons-2aef5174.4e92c599.js
commons-45c7c1ae.ae496366.js
commons-9b28023d.932a3bc9.js
commons-5cd0c127.20e132b9.js
commons-9c5b28f6.92f65076.js
commons-88a3c260.c8b40658.js
commons-cf0bbd8c.824a1ba4.js
commons-92a4fe01.4006388b.js
shared-6sI%2BDPbHATTXcp9U1qboQTGopM%3D-31ecd969.b46e6c57.js
shared-IWsrdoKm4A6gjoTLJxEjFp%2B8O0Y%3D-253ae210.84b3a325.js
FrontPage-253ae210.a558b8c6.js
FrontPage-45c7c1ae.c6cdda75.js
```

Reply:
As discussed with @jroebu14 this change actually fixes an issue where all our static assets `gziped` on top of each other into one `.gz` file.

I'm not sure the removal of the `-` is to do with this config here but the change isn't harmful.